src/derivatives/atlas.md

```diff
@@ -155,22 +150,22 @@ A guide for using macros can be found at
    "mni152nlin2009casym-pipeline": {
       "tpl-MNI152NLin2009cAsym": {
          "anat": {
-            "tpl-MNI152NLin2009cAsym_res-1_label-brain_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-eye_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-face_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-head_mask.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-CSF_probseg.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-GM_probseg.nii.gz": "",
-            "tpl-MNI152NLin2009cAsym_res-1_label-WM_probseg.nii.gz": "",
+            "atlas-MNI152NLin2009cAsym_res-1_seg-brain_mask.nii.gz": "",
+            "atlas-MNI152NLin2009cAsym_res-1_seg-eye_mask.nii.gz": "",
+            "atlas-MNI152NLin2009cAsym_res-1_seg-face_mask.nii.gz": "",
+            "atlas-MNI152NLin2009cAsym_res-1_seg-head_mask.nii.gz": "",
```

Comment:
I think tpl-MNI152NLin2009cAsym_res-1_label-brain_mask.nii.gz is valid, given that the label entity is valid (https://bids-specification.readthedocs.io/en/stable/glossary.html#label-entities), which belongs to the template. I see the dseg parts as atlases.

Reply:
Good point. I've reverted these changes.